### DISCUSSION TITLE
Just a couple questions cause I'm pretty confused on what to do ;-;

### DISCUSSION THREAD

**Discussion by Jankyjames43** [Monika-After-Story/MonikaModDev#9150] [QA - tech support + general] [answered]
First: is this meant to be for mobile or for computers/laptops or both. 
Secondly, I've downloaded the latest update (on mobile) this is my first time doing something like this and all its doing is showing me a bunch of file and not actually installing the game? 
 And also i've got a bunch of A, h and some thumbs as well as some checkbox's in my Google photos is that normal? And you know some tanktops and sweaters and hair models

**Comment by Booplicate** [comment, marked as answer]
The official mod works on Windows, Linux, and MacOS. We do **not** have android/ios builds. Whatever you downloaded, it's not this mod.
